Cubical/Categories/Monoidal/Braided.agda

```diff
@@ -0,0 +1,73 @@
+-- Braided Monoidal Categories
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Monoidal.Braided where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Constructions.BinProduct
+open import Cubical.Categories.Functor.Base
+open import Cubical.Categories.Functor.BinProduct
+open import Cubical.Categories.Morphism
+open import Cubical.Categories.NaturalTransformation.Base
+open import Cubical.Foundations.Prelude
+open import Cubical.Categories.Monoidal.Base
+
+module _ {ℓ ℓ' : Level} (C : Category ℓ ℓ') where
+  open Category C
+  open Functor
+
+  private
+    -- swaps the arguments of a bifunctor
+    Swap : Functor (C × C) C → Functor (C × C) C
```

Comment:
I think instead of this special-purpose function it would be better to compose `─⊗─` with a certain swapping functor of general type `Functor (C × D) (D × C)`, to be defined in `Cubical.Categories.Constructions.BinProduct`. This would incidentally also be the first step to resolve a todo note there: "- prove product is commutative up to isomorphism".